src/general/PauseMenu.cs

```diff
@@ -237,4 +299,13 @@ private void OnSwitchToMenu()
     {
         SceneManager.Instance.ReturnToMenu();
     }
+
+    private void ShowExitConfirmationDialog(string message)
+    {
+        exitConfirmationDialog.GetNode<Label>("DialogText").Text = message;
+        exitConfirmationDialog.PopupCenteredShrink();
+        exitConfirmationDialog.GetOk().ReleaseFocus();
+        exitConfirmationDialog.GetOk().Text = TranslationServer.Translate("YES");
+        exitConfirmationDialog.GetCancel().Text = TranslationServer.Translate("NO");
```

Comment:
Why are the buttons overridden? We haven't done this for other confirmation dialogs so this is inconsistent with them...

Reply:
A couple of comments above: https://github.com/Revolutionary-Games/Thrive/pull/2208#discussion_r669451372.

Reply:
That comment also applies to the loading and old save etc. dialogs that also have "OK", "Cancel" buttons by default. This type of dialog would be widely usable, so maybe this should be its own class and one that reacts to locale change.
Also the translation keys should probably be changed to "YES_AGREE", NO_DISAGREE" or something as YES and NO are too general in my opinion to be usable for all languages.

Reply:
touched upon once #2387 is done